ibm/provider.go

```diff
@@ -194,6 +194,7 @@ func Provider() terraform.ResourceProvider {
 			"ibm_storage_block":                         resourceIBMStorageBlock(),
 			"ibm_storage_file":                          resourceIBMStorageFile(),
 			"ibm_subnet":                                resourceIBMSubnet(),
+			"ibm_firewall_shared":                       resourceIBMFirewallShared(),
```

Comment:
Can we change resource name to ibm_hardware_firewall_shared
https://cloud.ibm.com/docs/infrastructure/hardware-firewall-shared?topic=hardware-firewall-shared-configuring-the-hardware-firewall-shared-